pkg/model/encoding.go

```diff
@@ -0,0 +1,21 @@
+package model
+
+import (
+	"github.com/grafana/tempo/pkg/tempopb"
+
+	"github.com/gogo/protobuf/proto"
+)
+
+// CurrentEncoding is a string representing the encoding that all new blocks should be created with
+const CurrentEncoding = ""
+
+// TracePBEncoding is a string that represents the original TracePBEncoding. Pass this if you know that the
+// bytes are encoded *tracepb.Trace
+const TracePBEncoding = ""
```

Comment:
Should we use a different value? e.g. `const TracePBEncoding = "pb"`

Reply:
This has to be an empty string b/c it represents the encoding that is running in Tempo now which will loads as an empty string:

https://github.com/grafana/tempo/pull/678/files#diff-a427eb1e5a0cc55b603ffc38455586344d3b33c778a17d201f01ea6c6241fed9R259

Reply:
Maybe we could return by default that encoding if there were less than 5 segments to the file name. It's a total nitpick anyway.

Reply:
It's a good point and was considered. The issue is that when we go to delete the file we call `fullFilename()` to get the file to delete and it would have to undo this logic or it won't find the file to delete correctly. We made this mistake when we first rolled out wal compression.

You are correct it would be nicer to have an actual string (instead of empty) to identify the original encoding. I'll think about it some.